packages/miniflare/src/index.ts

```diff
@@ -1903,6 +1956,9 @@ export class Miniflare {
 			// Remove from instance registry as last step in `finally`, to make sure
 			// all dispose steps complete
 			maybeInstanceRegistry?.delete(this);
+
+			// Close the inspector proxy server if there is one
+			await this.#maybeInspectorProxy?.dispose();
```

Comment:
Should this go above the `maybeInstanceRegistry?.delete(this)` call?
From the comment above that statement it suggests that we don't want to remove the instance from the registry until all the steps have completed.

Reply:
yeah, I think that's a good call 👍